### ISSUE TITLE
Longitud de líneas de los archivos .csv

### ISSUE THREAD

**Issue by leoWeitz** [PI-ITBA/TPE#14]
Buenas noches! Queríamos consultar si es posible asumir una longitud máxima para una línea cualquiera de los datasets.

Muchas gracias

**Comment by marcelogarberoglio** [comment]
Sí, pueden asumir una longitud máxima en base a los csv publicados, y un poco más por las dudas.
